fishtank-cli/src/commands/start.ts

```diff
@@ -15,14 +15,22 @@ export abstract class Start extends Command {
     },
   ]
 
+  static flags = {
+    noBootstrap: Flags.boolean({
+      char: 'B',
+      description:
+        'Do not start any bootstrap node and do not mine any block during the creation of the cluster',
```

Comment:
The description of noBootstrap flag should start with something like `start a network without bootstrap node`.

Reply:
What this does is to start a "cluster", not a "network". There's also the mining part that needs to be referenced: even if it's not currently implemented, it will be soon.

Reply:
The point is to have the description starting with describing what the flag does.

Reply:
Not sure I follow: this description says exactly what this flag does. It suppresses a behavior, hence it starts with "do not". I can reword it to "Start a cluster without a bootstrap node and without mining any block", but it wouldn't be accurate that this flags starts a network, so it wouldn't meet your goal.

Reply:
I said `something like`, it means approximately. You don't need to take it word by word